charts/portshift-operator/0.1.0/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: portshift-operator
+version: 0.1.0
+appVersion: v0.1.3
+description: |
+  Portshift cloud-native security platform is an agentless security solution for containerized applications
+keywords:
+  - portshift
+  - operator
+  - monitoring
+  - security
+  - alerting
+  - metric
+  - troubleshooting
+  - run-time
+home: https://www.portshift.io/
+icon: https://www.portshift.io/wp-content/uploads/2019/10/portshift-logo-68.png
```

Comment:
Ideally, this file should be local in the `portshift-operator` folder and referenced here as `file://../logo.png`. This way the logo can be used in airtight environments.